OWNERS

```diff
@@ -15,3 +15,4 @@ approvers:
   - derekwaynecarr
   - stevekuznetsov # for build and tooling changes
   - pweil-
+  - simo5 # auth team lead
```

Comment:
Most team leads aren't in this list.  It covers security, oc, apiserver, controllers, sdn, proxy, kubelet, and builders.  Is this because the packages are entangled?  We've made a lot of progress in the past two releases.

Reply:
Spoke with David offline, I will add Simo into the various auth places.